internal/validators/ottl/transform_spec_validator_test.go

```diff
@@ -2,947 +2,640 @@ package ottl
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Conditions: []string{"IsMatch(span.attributes[\"test\"], \"bar\")"},
-				},
-			},
-			isErrorExpected: false,
+			name:       "condition only",
+			conditions: []string{`span.name == "HTTP GET"`},
 		},
 		{
-			name: "[span context] valid transform spec with statement only",
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Statements: []string{"set(span.attributes[\"test\"], \"foo\")"},
-				},
-			},
-			isErrorExpected: false,
+			name:       "statement only",
+			statements: []string{`set(span.attributes["processed"], "true")`},
 		},
 		{
-			name: "[span context] valid transform spec with IsRootSpan() function",
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Statements: []string{"set(span.attributes[\"isRoot\"], \"true\") where IsRootSpan()"},
-				},
+			name: "multiple conditions and statements",
+			conditions: []string{
+				`span.name == "HTTP GET"`,
+				`span.status.code == 1`,
+			},
+			statements: []string{
+				`set(span.attributes["http_method"], "GET")`,
+				`set(span.attributes["processed"], "true")`,
 			},
-			isErrorExpected: false,
 		},
 		{
-			name: "[span context] valid statement but incorrectly used as a condition",
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Conditions: []string{"set(span.attributes[\"test\"], \"foo\")"},
-				},
-			},
+			name:       "uses IsRootSpan() function",
+			statements: []string{`set(span.attributes["isRoot"], "true") where IsRootSpan()`},
+		},
+		{
+			name:            "invalid context",
+			conditions:      []string{`span.name == "HTTP GET"`},
+			statements:      []string{`set(datapoint.attributes["http_method"], "GET")`},
 			isErrorExpected: true,
 		},
 		{
-			name: "[span context] valid condition but incorrectly used as a statement",
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Statements: []string{"IsMatch(span.attributes[\"test\"], \"bar\")"},
-				},
-			},
+			name:            "statement used as condition",
+			conditions:      []string{`set(span.attributes["processed"], "true")`},
 			isErrorExpected: true,
 		},
 		{
-			name: "[span context] invalid context",
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Conditions: []string{"IsMatch(datapoint.attributes[\"test\"], \"bar\")"},
-					Statements: []string{"set(datapoint.attributes[\"test\"], \"foo\")"},
-				},
-			},
+			name:            "condition used as statement",
+			statements:      []string{`span.name == "HTTP GET"`},
 			isErrorExpected: true,
 		},
 		{
-			name: "[span context] invalid function name in condition",
-			transforms: []telemetryv1alpha1.TransformSpec{
-				{
-					Conditions: []string{"IisMatch(span.attributes[\"test\"], \"bar\")"},
-					Statements: []string{"set(span.attributes[\"test\"], \"foo\")"},
-				},
-			},
+			name:            "invalid function in condition",
+			conditions:      []string{`sspan.name == "HTTP GET"`},
```

Comment:
here it is an invalid context and not invalid function, can you adjust it to use invalid function?